inspire/modules/migrator/__init__.py

```diff
@@ -0,0 +1 @@
+from .receivers import *
```

Comment:
Nope, but, if if it will execute the global part of the module, in order to compile it, it will register the signals etc.